packages/boltz-swaps/tests/utxo/claimReverse.spec.ts

```
@@ -0,0 +1,288 @@
+        privateKey: new Uint8Array(32).fill(1),
+        publicKey: new Uint8Array(33).fill(2),
+    },
+    preimage: PREIMAGE,
+    claimAddress: "bcrt1quser",
+    receiveAmount: 50_000,
+    lockupTxHex: "deadbeef",
+    ...overrides,
+});
+
+beforeEach(() => {
+    Object.values(mocks).forEach((m) => m.mockReset());
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)